app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
+
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
+        elif person.get("husband"):
+            Person.people[person["name"]].husband =\
+                Person.people[person["husband"]]
+    return [person for person in Person.people.values()]
```

Comment:
It is not true because for example, if someone calls it twice it only expects created list

Reply:
Could you please explain the potential problem in more detail?

Reply:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/65166203/234876660-ced0b4cb-918a-407a-93e0-7b3fef3b5866.png">
What do you expect from this code?
I expect that both calls return me a list with on argument. But your code will return the first time one and second time two